python/py-fmf/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        teemtee fmf 1.4.1
```

Comment:
we prefer PyPI as source when using Python packages - any reason not to use that instead of GitHub? If not, please change.

Reply:
I checked. fmf does not have sources available for download on PyPI, only a no-arch wheel. So I opted to get the sources from its GitHub repo tag instead.